docs/core/project-sdk/msbuild-props-desktop.md

```diff
@@ -52,16 +52,16 @@ The following table shows which elements and [globs](https://en.wikipedia.org/wi
 | Element               | Include glob                 | Exclude glob | Remove glob |
 |-----------------------|------------------------------|--------------|-------------|
 | ApplicationDefinition | App.xaml or Application.xaml | N/A          | N/A         |
-| Page                  | \*\*/\*.xaml                 | \*\*/\*.user; \*\*/\*.\*proj; \*\*/\*.sln; \*\*/\*.vssscc<br>Any XAML defined by *ApplicationDefinition* | N/A |
+| Page                  | \*\*/\*.xaml                 | \*\*/\*.user; \*\*/\*.\*proj; \*\*/\*.sln(x); \*\*/\*.vssscc<br>Any XAML defined by *ApplicationDefinition* | N/A |
```

Comment:
Will readers recognize the "*.sln(x)" correctly? Would it make sense to add an include file? I'd think it should state that both extensions are valid, and if we recommend taking action to convert existing projects.